docs/Language Feature Status.md

```diff
@@ -10,11 +10,12 @@ efforts behind them.
 
 | Feature | Branch | State | Developers | Reviewer | LDM Champ |
 | ------- | ------ | ----- | ---------- | -------- | --------- |
-| [Ranges](https://github.com/dotnet/roslyn/blob/features/range/docs/features/range.md) | [range](https://github.com/dotnet/roslyn/tree/features/range) | Prototype | [vsadov](https://github.com/vsadov) | [agocke](https://github.com/agocke) | [jaredpar](https://github.com/jaredpar) |
+| [Ranges](https://github.com/dotnet/roslyn/blob/features/range/docs/features/range.md) | [range](https://github.com/dotnet/roslyn/tree/features/range) | Prototype | [vsadov](https://github.com/vsadov) | [OmarTawfik](https://github.com/OmarTawfik) | [jaredpar](https://github.com/jaredpar) |
```

Comment:
❔  Can you not use the `@OmarTawfik` syntax here?

Reply:
Not sure if that is supported or not and can't find a reference to say it is / isn't.